pinot-server/src/main/java/org/apache/pinot/server/starter/helix/SendStatsPredicate.java

```diff
@@ -210,6 +211,8 @@ public synchronized void onInstanceConfigChange(List<InstanceConfig> instanceCon
         } else {
           LOGGER.warn("Send MSE stats is now disabled (problematic versions: {})", _problematicVersionsById);
         }
+      } else if (!_sendStats) {
+        LOGGER.info("Send MSE stats is still disabled (problematic versions: {})", _problematicVersionsById);
```

Comment:
This will also log when, for example, a minion joins the cluster if and only if we are not sending stats.

Given `_sendStats` should almost always be true, I think it is acceptable.